manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml

```diff
@@ -10,13 +10,11 @@ data:
     leaderElection:
       leaderElect: false
     profiles:
-    - schedulerName: scheduler-plugins-scheduler
+    - schedulerName: default-scheduler
```

Comment:
Note that `- schedulerName: default-scheduler` is not compatible with a secondary scheduler configuration as the default scheduler is also running out there.